packages/arb-validator-core/gobridge/goEthNodeGraph.go.old

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
The `.old` suggests that this file should be deleted

Reply:
removed - file pushed accidentally